prometheus-client/src/test/java/ai/apptuit/metrics/client/prometheus/PrometheusClientTest.java

```diff
@@ -217,6 +217,13 @@ public void testGetStepSize() throws Exception {
     assertEquals(7200, PrometheusClient.getStepSize(7501000000L));
   }
 
+  @Test
+  public void testCompleteTrailingSlashURL() throws Exception {
+    assertEquals(new URL("https://api.apptuit.ai/prometheus/"), PrometheusClient.completeTrailingSlashURL(new URL("https://api.apptuit.ai/prometheus")));
```

Comment:
Add a test case that make a request to mock server without trailing quote and validate the request actually worked
Test case gets mockserver.getURL(), removes trailing "/" and creates a client without the trailing quote and makes a test query

Reply:
done